config/initializers/translation.rb

```diff
@@ -17,6 +17,22 @@ def ignore_paths
   Dir['**/'] - Dir['app/**/']
 end
 
+def templates_data_for_default_org_proc
+  default_org_name = Rails.application.secrets.default_funder_name
```

Comment:
After https://github.com/portagenetwork/roadmap/pull/814 is merged, this will instead have to use `default_funder_id`.
A corresponding change will also have to be made on line 25 (`.where(org_id: Org.find_by!(name: default_org_name).id)`).